### PR TITLE
[V6r20] Bug Fix API

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Interfaces
FIX: classmethods should not have self! (Dirac.py)

ENDRELEASENOTES

### CODE REVIEW

**Comment by andresailer** [comment]
They also don't use `cls` so they should be `@staticmethod`s

**Comment by fstagni** [comment]
Indeed! updated

File: Interfaces/API/Dirac.py [hunk 1]
```diff
@@ -430,7 +430,7 @@ def __forceLocal(self, job):
     else:
       jdl = job
 
-    if not re.search('\[', jdl):
+    if not re.search(r'\[', jdl):
```
Comment (andresailer) [inline Interfaces/API/Dirac.py]: wouldn't `if '[' not in jdl:` be just as good, but not using the whole regexp machinery.

File: Interfaces/API/Dirac.py [hunk 2]
```diff
@@ -2586,7 +2585,7 @@ def __getJDLParameters(self, jdl):
 
     try:
       parameters = {}
-      if not re.search('\[', jdl):
+      if not re.search(r'\[', jdl):
```
Comment (andresailer) [inline Interfaces/API/Dirac.py]: Same `'[' not in jdl`

**Comment by atsareg** [comment]
Fine with me. Just apply the André's suggestion, please.

**Pull request merged**